### PR TITLE
Tech Notes Feb2024

### PR DESCRIPTION
**Pull request by SaminaHusain**
Closes https://github.com/tc39/Reflector/issues/520

### CODE REVIEW

File: meetings/2024-02/February-6.md [hunk 1]
```diff
@@ -0,0 +1,1371 @@
...
+
+Delegates: re-use your existing abbreviations! If you’re a new delegate and don’t already have an abbreviation, choose any three-letter combination that is not already in use, and send a PR to add it upstream.
+
+You can find Abbreviations in delegates.txt
+
+**Attendees:**
+| Name                   | Abbreviation | Organization      |
+| ---------------------- | ------------ | ----------------- |
+| Chris de Almeida       | CDA          | IBM               |
+| Linus Groh             | LGH          | Bloomberg         |
```
Comment (ljharb) [inline meetings/2024-02/February-6.md:13]: the attendee tables are removed before publishing

Comment (ctcpip) [reply meetings/2024-02/February-6.md:13]: that's not true

Comment (ljharb) [reply meetings/2024-02/February-6.md:13]: huh, you're right, i could have sworn they were, but i randomly clicked on some older ones and they're there. sorry for the noise.

**Review by ctcpip — APPROVED** [review]

**Comment by ctcpip** [comment]
thank you Samina!  🙏

**Review by acutmore — APPROVED** [review]

**Comment by ctcpip** [comment]
plan to merge this on Friday, allowing for any last-minute feedback

**Comment by ctcpip** [comment]
ah just noticed filenames need to fixed

**Pull request merged**